sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/README.md

```diff
@@ -161,14 +161,14 @@ builder.Services.AddOpenTelemetry().UseAzureMonitor(o =>
 
 ```C#
 builder.Services.AddOpenTelemetry().UseAzureMonitor();
-builder.Services.ConfigureOpenTelemetryTracerProvider((sp, builder) => builder.AddSource("MyCompany.MyProduct.MyLibrary"));
+builder.Services.ConfigureOpenTelemetryTracerProvider(builder => builder.AddSource("MyCompany.MyProduct.MyLibrary"));
```

Comment:
IIRC, we decided to use `builder.Services.AddOpenTelemetry().WithTracing()`

Reply:
That's not how I remember it. :)

Would you expect to see the examples like this?
```
builder.Services.AddOpenTelemetry()
        .WithTracing(x => x.AddSource(""))
        .UseAzureMonitor();
```

Reply:
@TimothyMothra Have you confirmed with this new API users will not run in to issues like https://github.com/Azure/azure-sdk-for-net/issues/36339?

Also, I can't recall but does the sequence matter here?
```
builder.Services.AddOpenTelemetry().UseAzureMonitor();
builder.Services.ConfigureOpenTelemetryTracerProvider(builder => builder.AddProcessor(new FooProcessor()));
```

versus

```
builder.Services.ConfigureOpenTelemetryTracerProvider(builder => builder.AddProcessor(new FooProcessor()));
builder.Services.AddOpenTelemetry().UseAzureMonitor();
``

Reply:
> That's not how I remember it. :)
> 
> Would you expect to see the examples like this?
> 
> ```
> builder.Services.AddOpenTelemetry()
>         .WithTracing(x => x.AddSource(""))
>         .UseAzureMonitor();
> ```

I proposed to use the below.

```
builder.Services.AddOpenTelemetry() .UseAzureMonitor();
builder.Services.AddOpenTelemetry() .WithTracing(x => x.AddSource(""));
```
There are three different APIs, and it's a bit confusing to recommend which one to use here. It would be better to get input from Blanch on which API to use.